### PR TITLE
Correct permissions for every view.

### PR DESCRIPTION
**Pull request by MurenMurenus**

### CODE REVIEW

File: ElephaHealth/chill_app/views.py [hunk 1]
```diff
@@ -23,24 +25,36 @@ def homepage(request: HttpRequest) -> HttpResponse:
...
+    serializer_class = UserProfileSerializer
+    permission_classes = [IsAdminUser]
+
+
+class UserProfileDetailView(ListCreateAPIView):
     serializer_class = UserProfileSerializer
     permission_classes = [IsOwnerProfileOrReadOnly, IsAuthenticated]
 
+    def get_queryset(self):
+        data = User.objects.filter(pk=self.request.user.pk)
```
Comment (TheBlek) [inline ElephaHealth/chill_app/views.py:46]: This could just be inlined, right?

File: ElephaHealth/chill_app/views.py [hunk 2]
```diff
@@ -23,24 +25,36 @@ def homepage(request: HttpRequest) -> HttpResponse:
...
-    permission_classes = [IsAuthenticated]
+    permission_classes = [IsAdminUser]
 
     def perform_create(self, serializer):
         user = self.request.user
         serializer.save(user=user)
 
 
-class UserProfileDetailView(RetrieveUpdateDestroyAPIView):
+class PkUserProfileDetailView(RetrieveUpdateDestroyAPIView):
```
Comment (TheBlek) [inline ElephaHealth/chill_app/views.py:35]: I feel like PkUserProfileDetailView is not the best name here. Maybe smth like AdminUserProfileDetailView or UserbaseProfileDetailView?

Comment (TheBlek) [reply ElephaHealth/chill_app/views.py:35]: Because the name does not convey that this gives all the users(As far as I can understand)

**Review by TheBlek — CHANGES_REQUESTED** [review]

File: ElephaHealth/chill_app/views.py (hunk 1 above)
Comment (MurenMurenus) [reply ElephaHealth/chill_app/views.py:46]: No, get_queryset() needs to be customized.

**Review by TheBlek — APPROVED** [review]

**Pull request merged**